packages/components-js/tests/e2e/playwright/specs/banner.e2e.ts

```diff
@@ -159,9 +155,9 @@ describe('close', () => {
       </p-banner>`
     );
 
-    const banner1 = await selectNode(page, '#banner1');
-    const banner2 = await selectNode(page, '#banner2');
-    const closeButtonBanner2 = await selectNode(page, '#banner2 >>> p-inline-notification >>> p-button-pure');
+    const banner1 = await page.$('#banner1');
+    const banner2 = await page.$('#banner2');
+    const closeButtonBanner2 = await page.$('#banner2 p-inline-notification p-button-pure');
```

Comment:
Looks like a nested light DOM selector but this works through shadow DOMs?
Perhaps there is another syntax to differentiate since this is a potential source for errors.

Reply:
We should use locators anyway but these also pierce through shadowDoms, thats the default behavior of playwright.

[https://playwright.dev/docs/locators#locate-in-shadow-dom](https://playwright.dev/docs/locators#locate-in-shadow-dom)